### PR TITLE
HW2 v1

### PR DESCRIPTION
**Pull request by 12Oleg**
Objective
Refactor an existing Node.js application created with Express.js and Swagger documentation to use TypeScript.

Requirements
    
1. Translating the Project to TypeScript:
        1. Create a copy of your existing Node.js project.
        2. Translate all server files and configurations to TypeScript.
        3. Install necessary packages for TypeScript development (e.g., ts-node, @types/express, etc.).
    
2. Preserve Functionality:
        1. Ensure all previous functionality, including the health-check endpoint and Swagger documentation, remains operational.
    
3. Typing:
        1. Add types to all variables, parameters, and functions in your application, including Express.js routes and Swagger specifications.
        2. Use types to ensure safety and provide hints during development.
    
4. Testing:
        1. Verify that your translated application still functions correctly, including the health-check endpoint and Swagger documentation.
        2. Use linters and other tools to check for errors.

### CODE REVIEW

File: src/config/openApiTypes.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+interface OpenApiOptions {
```
Comment (vasylkivskyy) [inline src/config/openApiTypes.ts]: You can import { Options } from "swagger-jsdoc", without creating an interface

File: package.json [hunk 2]
```diff
@@ -4,7 +4,11 @@
   "description": "node.js_HW",
   "main": "server.js",
```
Comment (vasylkivskyy) [inline package.json]: Is it correct way to file?

**Review by vasylkivskyy — CHANGES_REQUESTED** [review]

File: src/config/openApiTypes.ts (hunk 1 above)
Comment (12Oleg) [reply src/config/openApiTypes.ts]: I deleted interface OpenApiOptions and imported { Options } from "swagger-jsdoc". 
Thank you for sharing this knowledge!

File: package.json (hunk 2 above)
Comment (12Oleg) [reply package.json]: I have made corrections in the 'main': 'build/server.js'. Thank you!

**Review by vasylkivskyy — APPROVED** [review]

**Pull request merged**